Makefile

```diff
@@ -16,9 +16,13 @@ endif
 
 ifeq ($(detected_OS),Darwin)
  GOBIN_SHARED_LIB_EXT := dylib
+  # Building on M1 is still not supported, so in the meantime we crosscompile by default to amd64
   ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
-    # Building on M1 is still not supported, so in the meantime we crosscompile to amd64
-    GOBIN_SHARED_LIB_CFLAGS=CGO_ENABLED=1 GOOS=darwin GOARCH=amd64
+    USE_ARCH := amd64
+	ifneq ($(origin FORCE_ARCH),undefined)
+		USE_ARCH := $(FORCE_ARCH)
```

Comment:
Why can't you just use the `USE_ARCH` variable by itself and use lazy assignment?
```Makefile
USE_ARCH ?= amd64
```
https://www.gnu.org/software/make/manual/html_node/Flavors.html

Reply:
Good idea. Didn't know about the conditional variable assignment. Thanks for the heads up @jakubgs.